Provenance-Emu/Provenance
# Update openvgdb for real, this time

### What does this PR do
Updates the file openvgdb.sqlite to match the current checksum contained at https://github.com/OpenVGDB/OpenVGDB. This should fix artwork downloading.

### Where should the reviewer start
Build Provenance from source.

### How should this be manually tested
After building with this updated file, remove your image cache and then re-import roms from the app. Alternatively, using Files app or within the web interface (for uploading roms), delete the default.realm and corresponding lock file. Close and launch the app again. After roms are imported, close and reopen the app a second time. Box art should be downloaded/downloading.

### Any background context you want to provide
The current openvgdb checksum does not match what is in the latest official release. Additionally, looking at the current file with a text editor reveals URLs that are outdated for box art. This has been causing box art downloads to fail.

### What are the relevant tickets
#1314 #1264 
### Screenshots (if appropriate)

### Questions
